pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -50,7 +50,11 @@ type PipelineRunSpec struct {
 	// Params is a list of parameter names and values.
 	Params []Param `json:"params,omitempty"`
 	// +optional
-	ServiceAccount string `json:"serviceAccount"`
+	ServiceAccountName string `json:"serviceAccountName,omitempty"`
+	// DeprecatedServiceAccount is a depreciated alias for ServiceAccountName.
+	// Deprecated: Use serviceAccountName instead.
+	// +optional
+	DeprecatedServiceAccount string `json:"serviceAccount,omitempty"`
 	// +optional
 	ServiceAccounts []PipelineRunSpecServiceAccount `json:"serviceAccounts,omitempty"`
```

Comment:
Should those be `ServiceAccountNames` ? :stuck_out_tongue_closed_eyes:

Reply:
I couldn't decide if it was similar to ServiceAccountName -> ServiceAccount in v1.core or not.
Let me know if you want it switched.